library/src/main/kotlin/org/codice/compliance/Common.kt

```diff
@@ -13,12 +13,18 @@
  */
 package org.codice.compliance
 
+import com.sun.org.apache.xml.internal.serialize.OutputFormat
```

Comment:
✏️ Importing `com.sun.*` packages is generally frowned upon, as there are no guarantees what JVM will be run by an end user. Is there a reason these are being used and not some external, more general library?

Reply:
I will look for alternatives